src/Storage/Storage/Common/AzureStorageQueue.cs

```diff
@@ -98,7 +98,7 @@ public AzureStorageQueue(CloudQueue queue)
         }
 
         // Convert Track1 queue object to Track 2 queue Client
-        protected static QueueClient GetTrack2QueueClient(CloudQueue cloudQueue, AzureStorageContext context)
+        public static QueueClient GetTrack2QueueClient(CloudQueue cloudQueue, AzureStorageContext context)
```

Comment:
It looks this function is not used out side of [AzureStorageQueue.cs](https://github.com/Azure/azure-powershell/pull/21966/files#diff-8ec26b2a5e1a42c0015e613ca99d811ced1d8911e7d998b51256bd6bbc36b99f)

Reply:
Thanks for catching. I changed this for experiments and forgot to change it back, but this is not related to this cmdlet or useful. 
Reverted it back. 

RE: adding clientoptions to the function. Yes I'll add it in a later PR.